app/controllers/administrate/application_controller.rb

```diff
@@ -145,5 +153,9 @@ def show_search_bar?
         dashboard.collection_attributes
       ).any? { |_name, attribute| attribute.searchable? }
     end
+
+    def resources_json(resources, dashboard, resource_class)
+      resources.map { |resource| { name: dashboard.display_resource(resource), id: resource.send(resource_class.primary_key) } }
```

Comment:
So the Hound and `rubocop` is giving weird error messages for your branch.
Try rebasing from latest master and see what happens, there are few style errors which Hound should've pointed out, i.e this line is too long